### PR TITLE
Feature/energy cost calculation algorithm

### PR DESCRIPTION
**Pull request by Azuolas3**
- Added basic recipe model
- Created basic calculation of energy costs of a recipe based on the amount of minutes it takes and on what wattage.
- Prettier config
- Files "fixed" according to prettier
- Frontend for demonstration of energy cost calculation

### CODE REVIEW

File: ClientApp/src/components/EnergyCalculation.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from 'react';
+
+export default function EnergyCalculation() {
+  const [cost, getCost] = useState([]);
```
Comment (onlyidev) [inline ClientApp/src/components/EnergyCalculation.jsx:4]: Following the code standard used in previous commits this should be ``[cost, setCost]``

File: ClientApp/src/components/EnergyCalculation.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    const fetchData = async () => {
+      try {
+        const response = await fetch('energycost');
+        const number = await response.json();
+        console.log(number);
+        getCost(number);
+      } catch (error) {
+        console.log('error', error);
+      }
+    };
```
Comment (onlyidev) [inline ClientApp/src/components/EnergyCalculation.jsx:16]: This should be outside `useEffect()`

**Review by onlyidev — APPROVED** [review]
Code logic looks okay, there are minor code style errors in React.

File: ClientApp/src/components/EnergyCalculation.jsx (hunk 1 above)
Comment (Azuolas3) [reply ClientApp/src/components/EnergyCalculation.jsx]: Changed in new commit

File: ClientApp/src/components/EnergyCalculation.jsx (hunk 2 above)
Comment (Azuolas3) [reply ClientApp/src/components/EnergyCalculation.jsx]: Changed in new commit

**Review by Storas008 — APPROVED** [review]

**Pull request merged**